examples/FacebookTabsExample/FacebookTabBar.js

```diff
@@ -54,7 +55,6 @@ const FacebookTabBar = React.createClass({
             <Icon
               name={tab}
               size={30}
-              color={this.props.activeTab == i ? 'rgb(59,89,152)' : 'rgb(204,204,204)'}
```

Comment:
I don't want to remove this code. It is declarative representation of the state.